webcompat/form.py

```diff
@@ -118,9 +119,24 @@ def get_labels(browser_name):
 def normalize_url(url):
     '''normalize URL for consistency.'''
     url = url.strip()
-    if not url.startswith(SCHEMES):
+    parsed = urlparse.urlparse(url)
+
+    if url.startswith(BAD_SCHEMES):
```

Comment:
This won't do what we expect because `'http://'.startswith('http:/')` => `True`. Possibly we could do something with a RegExp or splitting on `//`?